pkg/cmd/start/main.go

```diff
@@ -133,6 +140,51 @@ func start(cmd *cobra.Command, args []string) {
 		log.Fatal(err)
 	}
 
+	// Create Service object to expose the metrics port.
+	operatorService, err := metrics.ExposeMetricsPort(ctx, viper.GetInt32("metrics-port"))
+	if err != nil {
+		log.Fatal(err)
+	} else if operatorService != nil {
+		if err := setOwnerReference(operatorService); err != nil {
```

Comment:
This is not needed we already do this in operator-sdk `metrics.ExposeMetricsPort` https://github.com/operator-framework/operator-sdk/blob/v0.8.1/pkg/metrics/metrics.go#L109

Reply:
@objectiser: I think you added this originally -- do you remember what problem it was solving?

Reply:
@jpkrohling The metrics service was not being removed.

Reply:
To test this, is it sufficient to not set the owner reference, create the operator, delete the operator, and check which services are available in the namespace?

Reply:
That should be adequate.

Reply:
Alright, I just confirmed it does remove the service when the operator is undeployed. I removed the extra code.

```console
$ kubectl get services
No resources found.
$ kubectl apply -f deploy/operator.yaml 
deployment.apps/jaeger-operator created
$ kubectl get pods
NAME                               READY     STATUS    RESTARTS   AGE
jaeger-operator-774d9c9f66-zxhz7   1/1       Running   0          16s
$ kubectl get services
NAME              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
jaeger-operator   ClusterIP   172.30.103.38   <none>        8383/TCP   14s
$ kubectl delete deployment jaeger-operator
deployment.extensions "jaeger-operator" deleted
$ kubectl get services
No resources found.
```